examples/flows/chat/chat-with-pdf/README.md

```diff
@@ -44,7 +44,7 @@ pf flow test --flow . --inputs question="What is the name of the new language re
 run_name="web_classification_"$(openssl rand -hex 12)
 
 # run with multiline data, --name is optional
-pf run create --flow . --data ./data/bert-paper-qna-3-line.jsonl --stream --column-mapping question='${data.question}' pdf_url='${data.pdf_url}' chat_history='${data.chat_history}' config='{ \"EMBEDDING_MODEL_DEPLOYMENT_NAME\": \"text-embedding-ada-002\", \"CHAT_MODEL_DEPLOYMENT_NAME\": \"gpt-35-turbo\", \"PROMPT_TOKEN_LIMIT\": \"2000\", \"MAX_COMPLETION_TOKENS\": \"256\", \"VERBOSE\": \"True\", \"CHUNK_SIZE\": \"256\", \"CHUNK_OVERLAP\": \"32\" }' --name $run_name
```

Comment:
save the original format since this is demonstration of manual input of multiple lines of data.

Reply:
or save one and change the other one.